akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -85,7 +88,7 @@ instead of modifying the existing instance, so while constructing long flows, re
 In the above example we used the ``runWith`` method, which both materializes the stream and returns the materialized value
 of the given sink or source.
 
-Since a stream can be materialized multiple times, the ``MaterializedMap`` returned is different for each materialization.
+Since a stream can be materialized multiple times, the materialized value returned is different for each materialization.
```

Comment:
I don't think it is worth to mention here. It just makes the introductory documentation harder to digest. Those elements which might materialize to the same value can explicitly say that in their API docs.